lib/fog/aliyun/storage.rb

```diff
@@ -97,6 +102,11 @@ def initialize(options = {})
           @port = uri.port || DEFAULT_SCHEME_PORT[@scheme]
 
           @persistent = options[:persistent] || false
+          @oss_client = OssClient.new(
+                                     :endpoint => endpoint,
```

Comment:
Does endpoint support https?

Reply:
The ruby sdk default scheme is http. But I have updated it and now it can support https.